<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menuAdminWindow</class>
 <widget class="QDialog" name="menuAdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1181</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tela Menu do Admin</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac" stdset="0">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1191</width>
     <height>601</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="groupBoxEstoque">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>321</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QListView" name="listView_Funcionarios">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>261</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_adicionarFuncionario">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>420</y>
       <width>33</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QScrollBar" name="verticalScrollBar_Funcionario">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>16</width>
       <height>401</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="searchLineEdit_Funcionarios">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Funcionários</string>
     </property>
    </widget>
    <widget class="QPushButton" name="refreshFuncionarios">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>15</y>
       <width>31</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>↻</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxVeiculos">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>321</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QListView" name="listView_Cleintes">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>261</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_adicionarCleintes">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>420</y>
       <width>33</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QScrollBar" name="verticalScrollBar_Veiculos">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>16</width>
       <height>401</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="searchLineEdit_Clientes">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Clientes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="refreshClientes">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>15</y>
       <width>31</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>↻</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxEstoque_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>30</y>
      <width>321</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QListView" name="listView_Estoque">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>261</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_adicionarEstoque">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>420</y>
       <width>33</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QScrollBar" name="verticalScrollBar_Estoque">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>16</width>
       <height>401</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="searchLineEdit_Estoque">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Estoque</string>
     </property>
    </widget>
    <widget class="QPushButton" name="refreshEstoque">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>15</y>
       <width>31</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>↻</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSair">
    <property name="title">
     <string>Sair</string>
    </property>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuSair"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
